{{cookiecutter.repo_name}}/{{cookiecutter.module_name}}/__init__.py

```diff
@@ -5,3 +5,64 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+# TODO: include relevant imports here
+# from .mycommand import my_command_function
+
+# Initialize the plugin
+plugin = QiitaPlugin('{{cookiecutter.project_name}}', '0.0.1',
+                     'Qiita Plugin: {{cookiecutter.project_name}}')
+
+# TODO: Define your commands. Here is an example on how to define one command.
```

Comment:
Should this also be in the README.md?

Reply:
It's here: https://github.com/qiita-spots/qp-template-cookiecutter/pull/2/files#diff-88b99bb28683bd5b7e3a204826ead112R68